ocs_ci/ocs/bucket_utils.py

```diff
@@ -2385,6 +2414,32 @@ def expire_objects_in_bucket(bucket_name, object_keys=[], prefix=""):
     )
 
 
+def expire_multipart_upload(upload_id):
+    """
+    Expire multipart upload in a bucket by changing their creation date to one year back.
+
+    Args:
+        bucket (str): The name of the bucket where the multipart upload parts reside
```

Comment:
upload_id** Rephrase in docstring

Reply:
Fixed in this commit: https://github.com/red-hat-storage/ocs-ci/pull/11756/commits/b2e227f01bdec882a39567c2f552a7e848e42c6a